airflow/providers/amazon/aws/example_dags/example_s3_to_redshift.py

```diff
@@ -41,13 +42,26 @@
     schedule_interval=None,
     tags=['example']
 ) as dag:
+    preparation__task_load_sample_data_to_s3 = PythonOperator(
+        python_callable=lambda: S3Hook().load_string("0,Airflow", f'{S3_KEY}/{REDSHIFT_TABLE}', S3_BUCKET),
+        task_id='preparation__load_sample_data_to_s3'
+    )
+    preparation__task_create_table = PostgresOperator(
+        sql=f'CREATE TABLE {REDSHIFT_TABLE}(Id int, Name varchar)',
+        postgres_conn_id='redshift_default',
+        task_id='preparation__create_table'
+    )
```

Comment:
That's what we usually tried to do in GCP, make example that is self-sufficient so I'm ok with this approach. However, this requires few things:
- operator must be idempotent ("setup" steps in example must work always)
- consistency between what is created in example and what is deleted in test tear down

Reply:
You are correct. But since I am `terraform destroy`ing the whole infra being used in `tearDown`. I don't need to remove the s3 object and the redshift table, do I?

Reply:
I do need to remove it, you are right. 👍 It should not depend on the `tearDown`.
And I will also make the setup steps idempotent. 👍